apps/throttles.py

```diff
@@ -9,37 +9,40 @@ class OrgBurst(SimpleRateThrottle):
     scope = 'org_burst'
 
     def get_cache_key(self, request, view):
-        ident = request.user.token.payload['organization_id'] or self.get_ident(request)
-        print(f'Ident: {ident}')
-
-        log_metric('transmission_api_throttling.info',
-                   tags={'method': 'api_call', 'code': 'API Throttle', 'module': __name__})
-
-        return self.cache_format % {
-            'scope': self.scope,
-            'ident': ident
-        }
-
-#
-# class OrgSustained(SimpleRateThrottle):
-#     """
-#     API rate limiter for req/days
-#     """
-#     scope = 'org_sustained'
-#
-#     def get_cache_key(self, request, view):
-#         ident = self.get_ident(request)
-#
-#         try:
-#             ident = request.user.token.payload['organization_id']
-#         except KeyError as exception:
-#             print(exception)
-#             pass
-#
-#         log_metric('transmission_api_throttling.info',
-#                    tags={'method': 'api_call', 'code': 'API Throttle', 'module': __name__})
-#
-#         return self.cache_format % {
-#             'scope': self.scope,
-#             'ident': ident
-#         }
+        if request.user.is_authenticated:
```

Comment:
Is there a more elegant way to do this? The tests were failing for unauthenticated users and I was having issues retrieving the organization ID from the request.

Reply:
I think it's fine to keep the user.is_authenticated check in the get_cache_key function. But you could combine those three nested `if` statements

Reply:
Good call. Done.